docs/elements.md

```diff
@@ -305,6 +307,38 @@ def generate(data):
 
 ---
 
+### `pl-excalidraw` element
+
+Draw a vector diagram using [excalidraw](https://github.com/excalidraw/excalidraw). Only manual grading is supported.
+
+![](elements/pl-excalidraw.png)
+
+**question.html**
+
+```html
+<p>Draw something else, with a starter diagram</p>
+
+<pl-excalidraw
+  name="vector"
+  file="starter.excalidraw"
+  file_dir="client_files_question_path"
+></pl-excalidraw>
+```
+
+#### Customizations
+
+| Attribute  | Type   | Default                      | Description                                                                            |
+| ---------- | ------ | ---------------------------- | -------------------------------------------------------------------------------------- |
+| `name`     | string | -                            | Unique name to identify the widget with. Drawing submissions are saved with this name. |
+| `file`     | string | -                            | Optional file to load as the starter diagram                                           |
+| `file_dir` | string | `client_files_question_path` | Directory to load `file` from.                                                         |
```

Comment:
Please look at the `directory` attribute on other elements. They typically accept values like `serverFilesCourse` and `clientFilesCourse`, e.g.:

https://github.com/PrairieLearn/PrairieLearn/blob/9dda505756eded11fde4a69561eeebdc915b1aee/apps/prairielearn/elements/pl-code/pl-code.py#L193-L198

Reply:
I went with a mix of what I could find from multiple elements. Perhaps this logic should be moved into the pl python module.